src/core/annotation.js

```diff
@@ -973,8 +1033,15 @@ class PopupAnnotation extends Annotation {
     let parentSubtype = parentItem.get('Subtype');
     this.data.parentType = isName(parentSubtype) ? parentSubtype.name : null;
     this.data.parentId = dict.getRaw('Parent').toString();
-    this.data.title = stringToPDFString(parentItem.get('T') || '');
-    this.data.contents = stringToPDFString(parentItem.get('Contents') || '');
+
+    if (parentItem.has('RT')) {
+      if ((parentItem.get('RT').name ||
+        AnnotationReplyType.REPLY) === AnnotationReplyType.GROUP) {
+        // If parent item is group reply type,
```

Comment:
Let's replace this comment with: "Subordinate annotations in a group should inherit the group attributes from the primary annotation." (taken from the spec)